src/main/java/uk/gov/hmcts/reform/sendletter/config/BankHolidaysConfig.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sendletter.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.sendletter.services.date.holidays.BankHolidaysClient;
+
+@Configuration
+public class BankHolidaysConfig {
+
+    @Bean
+    public BankHolidaysClient client(RestTemplate restTemplate) {
+        return new BankHolidaysClient(restTemplate, " https://www.gov.uk/bank-holidays");
```

Comment:
Best to have the URL in a config file. Also, it starts with a space here.

Reply:
Yes even I thought of it 😀